src/main/java/com/cloudbees/jenkins/support/filter/ContentFilter.java

```diff
@@ -90,4 +110,33 @@ static String filter(@NonNull ContentFilter filter, @CheckForNull String text) {
             return text;
         }
     }
+
+    /**
+     * Access the {@link ContentMappings} as Saveable.
```

Comment:
Not good form to link to something restricted, even though Javadoc will allow it since it knows nothing of `@Restricted` which is basically a workaround for lack of Java Module System restrictions. Better to discuss in terms of what this does.